app/components/TrackComponent/index.js

```diff
@@ -93,29 +138,69 @@ export function TrackComponent({ collectionName, artistName, imageUrl, trackName
 
       <StyledDescription>
         <If condition={!isEmpty(artistName)} otherwise={<StyledTitle>No artist name available</StyledTitle>}>
-          <StyledTitle italic={true}> {artistName} </StyledTitle>
+          <StyledTitle>
+            <StyledSpan> Artist name: </StyledSpan> {artistName}
+          </StyledTitle>
         </If>
 
         <If condition={!isEmpty(collectionName)} otherwise={<Paragraph>No collection name available</Paragraph>}>
-          <Paragraph> {collectionName} </Paragraph>
+          <Paragraph>
+            <StyledSpan> Collection name: </StyledSpan> {collectionName}
+          </Paragraph>
         </If>
 
         <If condition={!isEmpty(trackName)} otherwise={<Paragraph>No track name available</Paragraph>}>
           <Paragraph>
             <StyledSpan> Track name: </StyledSpan> {trackName}
           </Paragraph>
         </If>
+
+        <If condition={!isEmpty(country)}>
+          <Paragraph>
+            <StyledSpan> Country: </StyledSpan> {country}
+          </Paragraph>
+        </If>
+        <If condition={!isEmpty(kind)}>
+          <Paragraph>
+            <StyledSpan> Kind: </StyledSpan> {kind}
+          </Paragraph>
+        </If>
+        <If condition={!isEmpty(primaryGenreName)}>
+          <Paragraph>
+            <StyledSpan> Genre: </StyledSpan> {primaryGenreName}
+          </Paragraph>
+        </If>
+        <If condition={!isEmpty(wrapperType)}>
+          <Paragraph>
+            <StyledSpan> Wrapper Type: </StyledSpan> {wrapperType}
+          </Paragraph>
+        </If>
+
+        {trackTimeMillis && (
+          <Paragraph>
+            <StyledSpan> Duration: </StyledSpan> {Math.floor(trackTimeMillis / 60000)}:
+            {Math.floor(trackTimeMillis / 1000) % 60}s
+          </Paragraph>
+        )}
```

Comment:
Why do we need it twice ?
<Paragraph>
            <StyledSpan> Duration: </StyledSpan> 
            {Math.floor(trackTimeMillis / 1000) % 60}s
          </Paragraph>
          
          Can we do like the above ?

Reply:
This is going to show us the duration in minutes and seconds.